websites/V/Vizzy/presence.ts

```diff
@@ -0,0 +1,122 @@
+const presence = new Presence({
+		clientId: "968353669491871754"
+	}),
+	browsingTimestamp = Math.floor(Date.now());
+function getElementByXpath(path: string) {
+	return document.evaluate(
+		path,
+		document,
+		null,
+		XPathResult.FIRST_ORDERED_NODE_TYPE,
+		null
+	).singleNodeValue;
+}
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			smallImageKey: "logo_512",
+			startTimestamp: browsingTimestamp
+		},
+		{ pathname, search } = window.location;
+	switch (true) {
+		case pathname === "/": {
+			presenceData.details = "Viewing the homepage";
+			presenceData.largeImageKey = "home";
+			break;
+		}
+		case pathname === "/creations" || pathname === "/community": {
+			presenceData.details = "Viewing community creations";
+			presenceData.largeImageKey = "discover";
+			break;
+		}
+		case pathname === "/project": {
+			presenceData.largeImageKey = "discover";
+			presenceData.details = "Viewing a project";
+			presenceData.state = `Project: ${
+				getElementByXpath(
+					"/html/body/div[1]/div/main/div/div/div[2]/div/div[2]/div[1]/h4"
+				).textContent
+			}`;
```

Comment:
```suggestion
			presenceData.state =
				getElementByXpath(
					"/html/body/div[1]/div/main/div/div/div[2]/div/div[2]/div[1]/h4"
				).textContent;
```

Reply:
Can't keep the `Project:` part?

Reply:
No, you already mention the user is viewing a project in the presence details right above